testing/test_warnings.py

```diff
@@ -584,6 +584,19 @@ def test_group_warnings_by_message(testdir):
     assert result.stdout.str().count(warning_code) == 1
 
 
+@pytest.mark.filterwarnings("ignore::pytest.PytestExperimentalApiWarning")
+@pytest.mark.filterwarnings("always")
+def test_group_warnings_by_message_summary(testdir):
+    testdir.copy_example("warnings/test_group_warnings_by_message_summary.py")
+    result = testdir.runpytest()
+    result.stdout.fnmatch_lines(
+        ["test_group_warnings_by_message_summary.py: 120 tests with warnings"]
+    )
```

Comment:
This can certainly be hardened a bit (matching more lines).
It would still pass with all warnings being displayed for example.
There's `consecutive=True` by now.

Reply:
done, thanks